src/plugins/vis_type_timeseries/public/application/components/lib/label_date_formatter.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import moment from 'moment';
+
+export const labelDateFormatter = (label: string, dateformat?: string): string => {
```

Comment:
really really small nit:
```suggestion
export const labelDateFormatter = (label: string, dateformat = 'lll') => {
```
You could simplify this func (default param as string, not required returning type).
This would also simplify:
```
formattedLabel = moment(label).format(dateformat);
```

Reply:
Thank you dear! Updated! 🤗